server/store/sqlstore/migrations/000001_system_settings_table.down.sql

```diff
@@ -0,0 +1 @@
+SELECT 1;
```

Comment:
nit: could morph just treat an empty file as a no-op?

Reply:
Heh, actually having an empty file is only a problem for MySQL, Postgres will process those perfectly, so I'm making that change

Reply:
Done in 3225998f7452cd5132b3e3d0c556eb8c84d64ed1